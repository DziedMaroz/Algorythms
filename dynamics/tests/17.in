519270408
334
84843
09513988
4231265
7
848
11322912
2
14267339
36
6
6621724
5059159
2
593604
471106
14884
445
269300755
70997
2
7848809
3139
274645
049
81188227
556079138
8234182
5
09188142
4462
6
146
993
2689023
3392
16114
107428
058
9530
7035
467345088
4
60
53
64061545
549
9815829
2928
031001
71
0
8168010
388410108
34612
412003035
55
9
7910378
953
8
3
66457535
1732
812411058
4
75397328
67180
16347
57
66993335
0484736
40
9340702
486
79
29
323148
8781923
0256813
09002
25049
99321874
227468610
2611736
8975
216157
89
44001150
39
092
723954856
40265
883997
8923
3099727
06209
7689812
284141
77280416
262727
25771317
321320
6163587
18
2
0988759
38
7394184
5367
5
49
407
4214
3892066
407270060
1
58918
9242883
9750167
51196374
2228656
01
39
57532
97831
430000329
117637
345
87269
528048279
6800064
74
8778
0
2
5
71
94906
280
3761090
141779405
336313032
491660
46
87
52289333
3
735
4
07
7926875
305326
9292
34395
378
8913687
180679
58
2433992
856557
8290612
27314
45768639
0145501
3334
227147447
800828
23785
6297622
8742
5121667
6032465
8004226
5947607
721323
981
5
7502
41
6201161
824
19286
70
7297
0182102
1797
44115721
3
866
557
2995312
019
13158391
561
20240810
786869
91
77
058
58528773
0968
80444945
55346488
121042
859
143
988825
48670387
69
0794868
69
028
7
353197094
6499
2
8811223
847
3936640
02597
615314
78
51667801
6
6278710
61706301
067602523
9
38
8
411805172
58985263
673049
42158
8426
838169545
078157153
56896477
55118
22491
733568
534157323
48
5039589
6747888
37618814
9978
09257082
6269819
3197131
60741
927
52
615699256
03839360
7956489
33329081
95
973010630
618660334
7
8834067
2172
5350361
37613370
21255651
0
38
32190
929
4346002
243
4881573
5782981
45
067570
5352225
99294008
54559614
2280715
2
632
54
07573
51
481221
088227
8377
3377470
244152
745656848
392040
33
302680372
22771
20
212233934
36542813
53831
913187780
8347078
16
036
8477362
70301618
51
9
474936
8
1950
2935
236243
134938
4
7522
71
4820492
06
8
5911221
560477816
9
276804346
1
63
661097
404497
0113795
8031158
66908921
62260
1025946
8
01087
861172
