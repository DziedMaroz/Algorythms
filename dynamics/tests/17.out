5
51 927 0 40 8
